test/support/test_complex.h

```diff
@@ -26,7 +26,7 @@
 #include <cassert>
 
 #if !_PSTL_MSVC_LESS_THAN_CPP20_COMPLEX_CONSTEXPR_BROKEN
-#    define STD_COMPLEX_TESTS_STATIC_ASSERT(arg, msg) static_assert(arg, msg)
+#    define STD_COMPLEX_TESTS_STATIC_ASSERT(arg) static_assert(arg)
 #else
 #    define STD_COMPLEX_TESTS_STATIC_ASSERT(arg, msg) assert(arg)
```

Comment:
I believe this `#else` define also needs to remove `msg` parameter.

Reply:
Fixed.